plugin.video.nbcsnliveextra/addon.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.nbcsnliveextra" name="NBC Sports Live Extra" version="2020.11.6+matrix.1" provider-name="eracknaphobia">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.simplejson" />
```

Comment:
Can this be dropped?

Reply:
@eracknaphobia can you find some time to update this PR?

Reply:
yep